ui/imports/Utils.qml

```diff
@@ -2,8 +2,14 @@ pragma Singleton
 
 import QtQuick 2.13
 import "../shared/xss.js" as XSS
+import "./BigNumber/bignumber.js" as BigNumber
```

Comment:
Maybe it might be a good idea to create an npm-inited folder with a `package.json` instead of including a copy of the lib? Might need some build love though @michaelsbradleyjr thoughts?